apps/examples/testcase/le_tc/kernel/Kconfig

```diff
@@ -151,11 +151,6 @@ config TC_KERNEL_PTHREAD
 	default n
 	select MUTEX_TYPES
 
-config TC_KERNEL_ROUNDROBIN
```

Comment:
Could you check line 43? You should remove that line as well.

Reply:
done